src/main/resources/default/templates/biz/jobs/job.html.pasta

```diff
@@ -109,6 +109,35 @@
     </t:editForm>
 
     <script type="text/javascript">
+        sirius.ready(function () {
+            function onChange() {
+                const formData = sirius.getFormData(sirius.querySelector(".edit-form"));
+                sirius.getJSON("/job/params/___job.getName()", formData).then(handleParameterUpdates);
+            }
+
+            function handleParameterUpdates(json) {
+                for (const name in json.params) {
+                    const _elem = sirius.querySelector('.param-' + name);
+                    let param = json.params[name];
+                    if (param.visible) {
+                        _elem.removeAttribute("hidden");
+                    } else {
+                        _elem.setAttribute("hidden", true);
+                    }
```

Comment:
Optional: Sorry if this sounds snarky but what is the benefit of saving 3 characters here instead of just writing `_element` which is more readable to the human eye?

Reply:
It's actually 6*3 charactes 😲 changed it anyway :P